includes/Vendor/StoreListsFilter.php

```diff
@@ -96,12 +99,15 @@ public function sort_by_options() {
      */
     public function filter_pre_user_query( $args, $request ) {
         if ( ! empty( $request['stores_orderby'] ) ) {
-            $orderby = wc_clean( $request['stores_orderby'] );
+            $orderby         = wc_clean( $request['stores_orderby'] );
             $args['orderby'] = $orderby;
```

Comment:
Please, make this two lines in one since as the `$orderby` isn't used anywhere.

```php
$args['orderby'] = wc_clean( $request['stores_orderby'] );
```